recipes/invoke/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.12.2" %}
+
+package:
+  name: invoke
+  version: {{ version }}
+
+source:  
+  fn: invoke-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/i/invoke/invoke-{{ version }}.tar.gz
+  md5: f46192ea664325464a02f1c7f735e3cb
+
+build:
+  number: 0
+  script: python setup.py install
```

Comment:
There is no need for `build.bat` and `build.sh`. Just the options here will be fine.